### ISSUE TITLE
[BUG] [Formatter] djlint:off without closing djlint:on only works when no error codes specified

### ISSUE THREAD

**Issue by twolfson** [Riverside-Healthcare/djLint#655]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Linux Mint 20.2
 - Python Version (``python --version``): Python 3.8.10
 - djLint Version (``djlint --version``): djlint, version 1.28.0
 - template language: Django

## Issue
Hey y'all, I submitted #649 and you patched it super quick =D Sadly, it only half-patched my use case =/

i.e. For certain files I want to use the following at the top, without a closing `djlint:on`

```
{# djlint:off H030,H031 #}
```

You did an amazing job making this work for `{# djlint:off #}`, just not when it's also `H030,H031` or some other error code =/

## How To Reproduce
**Setup:**

index.html:

```django
{# djlint:off H030,H031 #}
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>Hello world!</title>
    </head>
    <body>
        Content
    </body>
</html>
```

**Expected behavior:**

```bash
djlint index.html
# Should see success, no error code
```

**Actual behavior:**

```bash
djlint index.html
#####
index.html
────────────────────────────────
H030 3:0 Consider adding a meta description. <html lang="en"> 
H031 3:0 Consider adding meta keywords. <html lang="en"> 

Linted 1 file, found 2 errors.
```

**Comment by twolfson** [comment]
The latest version works great! Thanks y'all!!

**Comment by malo-cayocca** [comment]
Hi @christopherpickering, would it be possible to re-open this issue, or would you rather have a new one ? I think the fix works for the linter, but not always for the formatter.

I have just removed  `require_pragma=true` from my config, and have run into a problem with the formatter modifying files which start with `{# djlint:off #}`, here is an example to reproduce in the online test version, with the default config : 

```
{# djlint:off #}
<img 
/>
```
gets transformed into
```
{# djlint:off #}
<img />
```

The issue is not systematic however, if we keep the initial example and modify it a bit, it remains untouched : 
```
{# djlint:off #}
<div><p>Welcome to djLint online!</p></div>
```
